packages/js/src/plugins/auctionHouseModule/operations/findBidsByPublicKeyField.ts

```diff
@@ -0,0 +1,87 @@
+import { Commitment, PublicKey } from '@solana/web3.js';
+import { UnreachableCaseError } from '@/errors';
+import { Metaplex } from '@/Metaplex';
+import { Operation, OperationHandler, useOperation } from '@/types';
+import { DisposableScope } from '@/utils';
+import { BidReceiptGpaBuilder } from '../gpaBuilders';
+import { AuctionHouse, Bid, toLazyBid } from '../models';
+import { AuctionHouseProgram } from '../program';
+import { toBidReceiptAccount } from '../accounts';
+
+// -----------------
+// Operation
+// -----------------
+
+const Key = 'FindBidsByPublicKeyOperation' as const;
+
+/**
+ * @group Operations
+ * @category Constructors
+ */
+export const findBidsByPublicKeyFieldOperation =
+  useOperation<FindBidsByPublicKeyFieldOperation>(Key);
+
+/**
+ * @group Operations
+ * @category Types
+ */
+export type FindBidsByPublicKeyFieldOperation = Operation<
+  typeof Key,
+  FindBidsByPublicKeyFieldInput,
+  Bid[]
+>;
+
+/**
+ * @group Operations
+ * @category Inputs
+ */
+export type FindBidsByPublicKeyFieldInput = {
+  type: 'buyer' | 'metadata';
```

Comment:
We could easily add `mint` to that list since we can transform get the metadata address from the mint address deterministically.

```suggestion
  type: 'buyer' | 'metadata' | 'mint';
```